webrtc.html

```diff
@@ -6042,7 +6042,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <p>The <dfn><code>stop</code></dfn> method stops the
               <a><code>RTCRtpTransceiver</code></a>. The sender of this
               transceiver will no longer send, the receiver will no longer
-              receive, and the <a>negotiation-needed flag</a> is set.</p>
+              receive, <code>receiver.track</code>'s "onended" event will
+              fire, and the <a>negotiation-needed flag</a> is set.</p>
```

Comment:
I suspect that we don't want to fire the "ended" event synchronously here. Since the script is the source of this action, I think we could just end the track directly (without any event).

Not introduced by this change, but should we replace the wording about setting the 'negotiation-neeed flag' and use "mark this connection as needing negotiation" instead?